.travis/install.sh

```diff
@@ -36,6 +36,9 @@ if [[ "$DARWIN" = true ]]; then
             curl -O https://bootstrap.pypa.io/get-pip.py
             sudo python get-pip.py
             ;;
+        bundle)
```

Comment:
I was having difficulty getting the application to build when using my system's python. I then found an SO answer from the py2app creator saying that a non-system python should be used: http://stackoverflow.com/questions/17597616/py2app-isnt-creating-a-standalone-version-of-my-app

But, it is definitely possible that the build mechanism was failing because of something on my system.